src/FocusApp.Client/MauiProgram.cs

```diff
@@ -71,6 +75,7 @@ private static IServiceCollection RegisterServices(this IServiceCollection servi
         {
             // Registered as a singleton so the timer is not reset by page navigation
             services.AddSingleton<ITimerService, TimerService>();
+            services.AddSingleton<Helpers.PopupService>();
```

Comment:
Since there's an interface already made for the PopupService, could we have its injection mimic the <ITimerService, TimerService> injection for consistency?